changelogs/fragments/714-infinity.yaml

```diff
@@ -0,0 +1,5 @@
+---
+minor_changes:
+  - postgres - add support for postgres ``infinity`` timestamps by replacing them with
+    ``datetime.min`` / ``datetime.max`` values
+    (https://github.com/ansible-collections/community.postgresql/pull/714)
```

Comment:
```suggestion
  - postgres - add support for postgres ``infinity`` timestamps by replacing them with ``datetime.min``/``datetime.max`` values (https://github.com/ansible-collections/community.postgresql/pull/714).
```

Reply:
@jbisabel why didn't you just commit the suggestion?

Reply:
My bad, I was unsure whether it would mark the commit as verified then, since it marked all the other commits as unverified when I used the UI to do the rebase after this morning's update on main. Just tried it with the other commit suggestion and it seems like it does mark those as verified.